terraform_provider_ibm_z_linuxone/ansible/README.md

```diff
@@ -0,0 +1,165 @@
+# Manage z/OS Logical Partions with GitOps with Terraform Enterprise, Ansible Automation Platform, and Event-Driven Ansible
```

Comment:
Manage z/OS Logical Partions using GitOps with Terraform Enterprise, Ansible Automation Platform, and Event-Driven Ansible